blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitUtils.java

```diff
@@ -138,6 +138,9 @@ static List<ErrorMessage.Error> validateCredentials(@NonNull String uri, @Nullab
     private static boolean checkCauseNotAuthenticated( GitException e) {
        if(e.getCause() instanceof TransportException){
            TransportException te = (TransportException)e.getCause();
+           if(StringUtils.contains(te.getMessage(), "Cannot log in ")) {
```

Comment:
rely on error message :( 
so error message is not anymore from trilead but from apache mina so it;s a different one.